docs/cli.rst

```diff
@@ -259,6 +260,21 @@ or set its nodata value to, e.g., `0`:
 
     $ rio edit-info --nodata 0 example.tif
 
+or set its color interpretation to red, green, blue, and alpha:
+
+.. code-block:: console
+
+    $ rio edit-info --colorinterp 1=red,2=green,3=blue,4=alpha example.tif
```

Comment:
@geowurster my last question: would you be willing to consider a syntax change here?

rio-edit-info already allows us to set units and descriptions of selected bands like

`rio edit-info --bidx 1 --description "Band 1"`

I'd prefer if color interpretation were done more like that. For example:

`rio edit-info --bidx 1 --colorinterp red`

and for multiple bands in one go

`rio edit-info --bidx 1,2,3 --colorinterp red,green,blue`

I think we absolutely need to keep the shortcuts. That would be my go-to usage.

Reply:
@sgillies That makes commands like `$ rio edit-info --bidx 1,2,3 --description whatever --colorinterp red,green,blue` difficult.  What about dropping the `bidx=ci` syntax and just using `--colorinterp red,green,blue` and requiring the user to specify color interpretations for every band like `ds.colorinterp = [ColorInterp.red, ColorInterp.green, ColorInterp.blue]`?  The `bidx=ci` syntax is a hold over from when I was initially thinking about this in terms of a band mapping rather than a sequence.

Reply:
@geowurster I like dropping the `bidx=ci` syntax. How about the option to "broadcast" a single value across all bands like `--colorinterp grey` ? I'm happy to do that in a different PR.

Reply:
@sgillies Switched syntaxes.  Lets handle broadcasting in a different PR.